OPHD/UI/PopulationPanel.cpp

```diff
@@ -81,7 +81,7 @@ PopulationPanel::PopulationPanel() :
 
 	largestStringLength += edgeBuffer + mFont.width("999999");
 
-	mPopulationPanelWidth = mFontBold.width(constants::PopulationBreakdown) + edgeBuffer;
+	mPopulationPanelWidth = mFontBold.width(constants::PopulationBreakdown) + edgeBuffer + mFont.width(" : 999");//mFont.width(" : 999") Accounts for the space needed for worker/scientist avialibility.
 	const int windowWidth = largestStringLength + edgeBuffer + mPopulationPanelWidth;
```

Comment:
This might need some adjustment. Although the values are currently aligned with the end of the title string, it's a bit odd to be adding the length of values to the length of the title. There's an implied assumption about that alignment, which could potentially change in the future if the display is changed.

It would seem more intuitive to be adjusting the `largestStringLength` field instead. Though I may need to look at this a bit more closely to know if that's appropriate.

Perhaps a deeper question is if the colon separated double value will be clear. Maybe we need a new line with descriptive text to distinguish between total people and available people.

Reply:
I had to do a double take when you pointed this out. Why did I add it to mPopulationPanelWidth instead of largestStringLength? 

largestStringLength controls the size of the moral breakdown side of the population panel making room for the largest moral issue string and adding to it doesn't help make space for the : and worker usage availability on the population breakdown side. Adding to largestStringLength just makes the moral breakdown side larger. 

`const int windowWidth = largestStringLength + edgeBuffer + mPopulationPanelWidth;`

I didn't know why windowWidth isn't something like mMoralPanelWidth + edgeBuffer + mPopulationPanelWidth

For the deeper question I suppose we need to decide on what we want to indicate what the other value is. I just wanted some other number that i could intuit what the number of remaining workers and scientists instead of running out unexpectedly.

Reply:
Ahh, it seems I didn't examine the code closely enough. It seems you are right that's for the other side of the panel.

You might guess that I didn't design the UI, so I sometimes miss some of the fine details like that. Most of the UI code I've touched has been for low level refactoring, without much consideration for high level design.

I'm open to suggestions to rename things for better clarity.

Looking at the game UI, I'm thinking maybe we should add another section below the "Population Breakdown" section for "Available Population" (or some similar description), which shows the available number of workers and scientists. The other population components don't really contribute work, so they wouldn't need to be listed there. That might provide more clarity than a ":" separate set of two values.